### ISSUE TITLE
Unpin chrome version used for integration tests when chrome 122 releases.

### ISSUE THREAD

**Issue by jye-sf** [salesforce/lwc#3987]
### Description
Saucelabs Chrome browser version has been pinned to chrome 120 to bypass an issue with `shadowRoot` on chrome 121. That issue is fixed in chrome 122, which should roll out between 2/14 and 2/20.

At that point, we should unpin the chrome version. https://github.com/salesforce/lwc/pull/3986 should be reverted and the CI tests verified.

**Comment by nolanlawson** [comment]
@jye-sf This is only for `spring24` branch correct? Due to https://github.com/salesforce/lwc/pull/3986

**Comment by jye-sf** [comment]
@nolanlawson Yes, that's correct. I've added that detail to the main description.
